cli/Main.d.ts

```diff
@@ -3,6 +3,7 @@ type ErrorMessage = string;
 
 interface Main {
   renderers: Renderer[];
+  rendererForLanguage(language: string): Maybe<Renderer>;
```

Comment:
Now that we distribute as normal modules, you can define `rendererForLanguage` in `Language/Renderers.purs` to avoid bloat in `Main`.